Makefile

```diff
@@ -861,8 +861,8 @@ pkgs: build-tools $(PKGS)
 pkg/kernel:
 	$(QUIET): $@: No-op pkg/kernel
 
-pkg/external-boot-image/build.yml: pkg/external-boot-image/build.yml.in
-	$(QUIET)tools/compose-external-boot-image-yml.sh $< $@ $(shell echo ${KERNEL_TAG} | cut -d':' -f2) $(shell $(LINUXKIT) pkg show-tag pkg/xen-tools | cut -d':' -f2)
+pkg/external-boot-image/build.yml: pkg/external-boot-image/build.yml.in FORCE
```

Comment:
Why is this always FORCE?

Reply:
It's explained in the commit message:

```
The target that generates the pkg/external-boot-image/build.yml file must
be executed always in order to update the kernel tag. Otherwise, two
consecutive builds for amd64 and arm64 will fail:

make ZARCH=amd64 HV=kubevirt pkg/external-boot-image

It's going to generate pkg/external-boot-image/build.yml for amd64.

make ZARCH=arm64 HV=kubevirt pkg/external-boot-image

It's going to fail because the build.yml for amd64 it's already there and
it will not be re-generated for arm64.
```

Reply:
I assume each generation is a different content for `build.yml`, otherwise there would not be a need to rebuild it? If so, then shouldn't we have a different filename?

Reply:
actually I didn't give a good example, it's not actually about amd64 vs arm64, build.yml depends basically on two things:
1. pkg/xen-tools (so it can use the last TAG)
2. KERNEL_TAG : Here is the issue, we don't have a way to know if the TAG was updated or not (since users can specify a custom image through `make KERNEL_TAG=...`, so that's why we need to always update the build.yml file... 

I've updated the Makefile to include the pkg/xen-tools as dependency and updated the commit message as well....